### ISSUE TITLE
Update Getting Started steps for Intel (x86_64) macOS

### ISSUE THREAD

**Issue by hansemro** [ngscopeclient/scopehal-docs#79]
Current instructions are tailored for Apple Silicon Macs with homebrew installed at `/opt/homebrew` and ffts not required. 

According to https://docs.brew.sh/Installation, homebrew installs to `/usr/local/` on x86_64 Macs by default. To correct for this, we can obtain the `HOMEBREW_PREFIX` from `brew config` and update steps where affected.

Steps for ffts:
```
git clone https://github.com/anthonix/ffts.git
cd ffts
mkdir build
cd build
# TODO: replace /usr/local with HOMEBREW_PREFIX
cmake .. -DENABLE_SHARED=ON -DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_MACOSX_RPATH=1
make -j4 
sudo make install
```

**Comment by d235j** [comment]
Should be fixed by #81 (to use `$(brew --prefix)`).

**Comment by d235j** [comment]
@hansemro can you check if this is fixed and close if so?
